pom.xml

```diff
@@ -28,6 +28,7 @@
     <module>model</module>
     <module>server</module>
     <module>testing</module>
+    <module>log</module>
```

Comment:
I'm not sure this needs to be a separate module -- it could just be part of the server/ module.  It doesn't matter a ton, but it's one more POM to worry about.